views/address.tmpl

```diff
@@ -56,24 +56,38 @@
                         </tr>
                         {{end}}
                     {{else}}
-                    <tr>
-                        <td class="pr-2 lh1rem vam text-right xs-w91">UNCONFIRMED</td>
-                        <td class="fs28 mono nowrap fs16-decimal d-flex align-items-center justify-content-end">
-                            {{.NumUnconfirmed}}
-                        </td>
+                    <tr  class="h2rem">
+                        <td class="pr-2 lh1rem vam text-right xs-w91">TOTAL UNSPENT</td>
```

Comment:
TOTAL UNSPENT is in the code twice here because the {{if .Fullmode}} loop is covering an unnecessarily large area. Should only be covering the code that is needed and not duplicating the output.